src/main/java/seedu/address/MainApp.java

```diff
@@ -145,6 +148,9 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
         UserPrefs initializedPrefs;
         try {
             Optional<UserPrefs> prefsOptional = storage.readUserPrefs();
+            if (!prefsOptional.isPresent()) {
+                logger.info("Prefs file not found. Will be starting with a new default prefs file.");
```

Comment:
Why the phrasing of this is different from the config file message?

Reply:
My bad, I think I might have messed up during the rebasing, thanks for pointing it out. Really need to be more careful.